cd/application-deployment/dev/vaec-api-task-definition.json

```
@@ -327,8 +327,8 @@
   "requiresCompatibilities": [
     "FARGATE"
   ],
-  "cpu": "512",
-  "memory": "1024",
+  "cpu": "2048",
```

```suggestion
  "cpu": "1024",
```
I shot for the moon to eliminate it as a problem. We can pull this back to a more reasonable value.

---

AWS docs:
https://docs.aws.amazon.com/AmazonECS/latest/userguide/fargate-task-defs.html#fargate-tasks-size